### PR TITLE
add bottom margin to event tags in event cards  #2346

### PR DESCRIPTION
**Pull request by khushishikhu**
i have tried to resolve this issue #2346 . please guide me where i am not correct.

### CODE REVIEW

**Comment by khushishikhu** [comment]
Capture

there is a problem i am facing that....i am unable create event in my local machine......please help me why is this problem occurring with me.....i have already gone through the Read.me . But it is still there

**Comment by snitin315** [comment]
Maybe you haven't filled a required  field?

**Comment by khushishikhu** [comment]
@snitin315  sir i have filled all the fields and even tickets but i am still struck on this. please help me out what other thing i need to look for?

**Comment by iamareebjamal** [comment]
What's the error you're getting?

**Comment by khushishikhu** [comment]
sir i am unable to publish event in my localhost. please help me to how i can publish it.

**Comment by iamareebjamal** [comment]
What's the error you're getting?

**Comment by khushishikhu** [comment]
sir when i click on publish button then nothing is happening.

**Comment by iamareebjamal** [comment]
It is disabled. Check the form, something must be missing

**Comment by khushishikhu** [comment]
sir i have checked the form minutely but it is still disabled. even neither of the three button is working.

**Comment by iamareebjamal** [comment]
Enter the address and you'll be able to publish the event

**Comment by khushishikhu** [comment]
sir please review the changes. i have tried to fix this issue #4539

**Comment by khushishikhu** [comment]
sir please review it. and guide me what i need to do.

**Comment by iamareebjamal** [comment]
Please post the screenshot whenever there is a UI change

**Comment by khushishikhu** [comment]
Capture1
sir this was the error. the tags are going out of the card

**Comment by khushishikhu** [comment]
Capture2
i have tried to fix it. now the tags are not going out and even responsive.

File: app/styles/partials/footer.scss [hunk 1]
```diff
@@ -39,3 +39,7 @@ footer {
     right: 0 !important;
   }
 }
+
+  .ui.card > .content {
+  padding: 0.35rem;
+  }
```
Comment (iamareebjamal) [inline app/styles/partials/footer.scss:45]: This will add padding to all cards, not just event cards

Comment (khushishikhu) [reply app/styles/partials/footer.scss:45]: so i need to fix it only in event cards.

**Comment by khushishikhu** [comment]
Capture1

**Comment by khushishikhu** [comment]
Capture2
sir these are the ui changes i have done. now the padding is only in event cards. please review it and guide me to fix it

File: app/templates/index.hbs [hunk 2]
```diff
@@ -11,7 +11,7 @@
   {{/if}}
   {{#if this.filteredEvents}}
     <h2 class="main-heading">{{t 'Upcoming Events'}}</h2>
-    <div class="ui stackable three column grid">
+    <div class="ui stackable three column grid" id="event">
```
Comment (iamareebjamal) [inline app/templates/index.hbs:14]: id should be unique and not repeated

File: app/templates/index.hbs [hunk 3]
```diff
@@ -11,7 +11,7 @@
   {{/if}}
   {{#if this.filteredEvents}}
     <h2 class="main-heading">{{t 'Upcoming Events'}}</h2>
-    <div class="ui stackable three column grid" id="event">
+    <div class="ui stackable three column grid" id="uimargin">
```
Comment (iamareebjamal) [inline app/templates/index.hbs:14]: Id should be unique and not repeated

Comment (khushishikhu) [reply app/templates/index.hbs:14]: sir can you please help what type of id should i make. so that it's unique

File: app/templates/index.hbs [hunk 4]
```diff
@@ -11,7 +11,7 @@
   {{/if}}
   {{#if this.filteredEvents}}
     <h2 class="main-heading">{{t 'Upcoming Events'}}</h2>
-    <div class="ui stackable three column grid">
+    <div class="ui stackable three column grid" id="upcoming-event-card">
```
Comment (iamareebjamal) [inline app/templates/index.hbs:14]: ID shouldn't be used as ID needs to be unique and not repeated

**Comment by khushishikhu** [comment]
sir can you please guide me what other thing i could do to make margin in just event cards. As id can't be used.

**Comment by iamareebjamal** [comment]
What about a class?

**Comment by khushishikhu** [comment]
Capture1

**Comment by khushishikhu** [comment]
Capture2

@iamareebjamal sir i have tried to make it through class in just event cards only. please review it

File: app/templates/index.hbs [hunk 5]
```diff
@@ -11,6 +11,7 @@
   {{/if}}
   {{#if this.filteredEvents}}
     <h2 class="main-heading">{{t 'Upcoming Events'}}</h2>
+    <div class="upcoming-event">
```
Comment (iamareebjamal) [inline app/templates/index.hbs:14]: Don't create a new div wrapper

Comment (iamareebjamal) [reply app/templates/index.hbs:14]: Neither does this solve the issue

Comment (khushishikhu) [reply app/templates/index.hbs:14]: sir, then where i could add a class?

**Comment by snitin315** [comment]
@khushishikhu, This div is responsible for `event-card footer`. Try adding a Semantic UI class here. -
https://github.com/fossasia/open-event-frontend/blob/11b19c9be8b6ca24095a25db09b9dff0c190ae1d/app/templates/components/event-card.hbs#L33

File: app/templates/index.hbs [hunk 6]
```diff
@@ -11,16 +11,14 @@
   {{/if}}
   {{#if this.filteredEvents}}
     <h2 class="main-heading">{{t 'Upcoming Events'}}</h2>
-    <div class="upcoming-event">
-    <div class="ui stackable three column grid">
+    <div class="ui stackable three column grid upcoming-event">
```
Comment (iamareebjamal) [inline app/templates/index.hbs:14]: The issue is not in upcoming event section. It's in the event card. Read the issue again

**Comment by khushishikhu** [comment]
@snitin315  sir but what class i should add here and it will make changes in all the cards, not just event cards. please guide me what should i do.

**Comment by khushishikhu** [comment]
Capture1
sir can this be done?

**Comment by iamareebjamal** [comment]
What code changes were done, please push those so they can be reviewed

**Comment by iamareebjamal** [comment]
Use a class

**Comment by khushishikhu** [comment]
Capture1

**Comment by iamareebjamal** [comment]
The issue is still there in the screenshot

**Comment by iamareebjamal** [comment]
The changes will be applied to all cards, not only event cards

**Pull request closed without merging**